lib/routes/home_route/tabs_route.dart

```diff
@@ -215,10 +209,14 @@ class TabsRouteState extends State<TabsRoute> with TickerProviderStateMixin, Sel
                 filterFavorite: FavoritesControl.instance.showOnlyFavorites,
               ),
             ),
-            searchButton
+            const _ShowOnlyFavoritesButton(),
+            searchButton,
           ]
         : [
-            DeleteSongsAppBarAction<Content>(controller: selectionController),
+            DeleteSongsAppBarAction<Content>(
+              controller: selectionController,
+              keepSpacingWhenHidden: true,
```

Comment:
I just realized that I was trying to workaround a problem instead of fixing it - the AppBar actions had wrong alignment, fixed it in https://github.com/nt4f04uNd/nt4f04unds_widgets/pull/11/commits/b901b9988fb08ff58b8f4964dbe8b341433c972b

Deleted `keepSpacingWhenHidden`